src/api/ops-transformations.test.js

```diff
@@ -225,7 +231,7 @@ test('expand (single item)', t => {
   const op1 = cube()
   const obs = expand(10, 5, op1)
 
-  t.deepEqual(obs.polygons[0].vertices[0], {pos: {_x: -10, _y: 0, _z: 0}, tag: 25909})
```

Comment:
This test failed caused by a differing tag value. I don't excatly know what a tag is used for but it looks like some kind of uniqe value to me. Hope changing it doesn't have side effects and is ok.

Reply:
That is an unfortunate side effect for the internals of CSG objects (there is an ever increasing `tag` counter that is used by some of the boolean operation algorithm to distinguish or fuse together polygons. It is very clunky and quite annoying for tests since the tag value can vary depending on what tests ran before. 
So no worries :)